pkgs/shells/zsh/zsh-fzf-history-search/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, substituteAll, pkgs, bash, fzf }:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, bash
, fzf
}:
```
One item per line is preferable for future diff.